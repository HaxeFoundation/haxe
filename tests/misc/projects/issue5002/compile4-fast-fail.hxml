compile4-fail.hxml
-D fail-fast
