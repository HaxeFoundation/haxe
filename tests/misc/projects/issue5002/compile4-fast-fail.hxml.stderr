Main4.hx:3: characters 7-9 : Variable names starting with a dollar are not allowed: "$i"
